dj_rql/filter_cls.py

```diff
@@ -268,6 +268,16 @@ def build_q_for_filter(self, data):
         )
         django_lookup = self._get_django_lookup(filter_lookup, str_value, null_values)
 
+        django_field = base_item.get('field')
+        use_repr = base_item.get('use_repr', False)
+
+        typed_value = None
+        if django_field is not None:
+            typed_value = self._get_typed_value(
```

Comment:
It does seem to make sense to pass `typed_value` somehow to `build_q_for_custom_filter` as otherwise it will be required to duplicate calculations there. I guess, the best way is to go with kwargs in FilterArgs, but may be some other options. Do you agree?

Reply:
Agreed, let's leave as is.